drivers/w1/Kconfig.ds248x

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2022, Caspar Friedrich <c.s.w.friedrich@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config W1_DS248X
+	bool "DS2482/4 Single-Channel 1-Wire Master"
```

Comment:
You need to add something like:

`DT_COMPAT_MAXIM_DS248X := maxim,ds248x`

Reply:
I see. I somehow expected that this is something automatic. I wasn't familiar with this structure. Going to integrate this later.

Reply:
I'm looking to make that a bit more regulator, with https://github.com/zephyrproject-rtos/zephyr/pull/47196, but will see how that progresses.

Reply:
Good idea! Fixed.